wagtail/admin/templates/wagtailadmin/notifications/task_state_submitted.html

```diff
@@ -0,0 +1,12 @@
+{% extends 'wagtailadmin/notifications/base.html' %}
+
+{% load i18n %}
+
+{% block content %}
+    <p>{% blocktrans with task=task.name %}The page "{{ page }}" has been submitted for approval in moderation stage "{{ task }}".{% endblocktrans %}</p>
```

Comment:
Looks like this (and the plain text version) is missing a `page=page.get_admin_display_title` in the blocktrans tag.

Reply:
I've gone through and added these - thanks.